db/seeds.rb

```diff
@@ -19,7 +19,7 @@ def random_record(klass)
 end
 
 def random_record_for_org(org, klass)
-  klass.where(organization: org).limit(1).order(Arel.sql('random()')).first
+  klass.where(organization: org).all.sample
```

Comment:
Because we're in fresh-seeding territory and row counts are very low, we went with `all.sample` which is pretty foolproof.